### DISCUSSION TITLE
普通书籍，报纸的录入

### DISCUSSION THREAD

**Discussion by banned-historical-archives** [banned-historical-archives/banned-historical-archives.github.io#399] [Q&A] [answered]
像文稿一样按文章标题拆出来？
与文稿放在一个表格？

**Comment by PirateBook1** [comment, marked as answer]
普通书籍可以按章节存储，“章节”是广义概念，其实就是按照目录把书拆成几部分

报纸的话，里面每篇文章其实也是一种“章节”

表的结构直接用目前的，不过最好另开一张表不要合并进目前的文稿里

**Reply by banned-historical-archives** [reply]
文稿里面也有来自报刊的文章

**Reply by PirateBook1** [reply]
那么，索性全部合到一起？

**Reply by PirateBook1** [reply]
这样的话问题又回到搜索上了

**Reply by banned-historical-archives** [reply]
我觉得合并比较好，可以对比

**Comment by PirateBook1** [comment]
@banned-historical-archives 你试着解析《红旗》吧，看看效果怎么样

**Reply by PirateBook1** [reply]
现在资料库里有录进来的书籍吗

**Reply by PirateBook1** [reply]
我看看效果

**Reply by banned-historical-archives** [reply]
还没有，红旗的排版不好处理

**Reply by banned-historical-archives** [reply]
![图片](https://user-images.githubusercontent.com/109972625/195319233-46f76db6-531d-4d32-af58-8a68790c0b95.png)

**Reply by banned-historical-archives** [reply]
需要多加一个ocr参数，force_vsplit

**Reply by banned-historical-archives** [reply]
现在可以了, 你参考一下 #440

**Reply by PirateBook1** [reply]
书籍录入的一般流程是要每篇文章都依次手动输入吗

**Reply by banned-historical-archives** [reply]
恩

**Reply by PirateBook1** [reply]
其实不如把 pdf 全部拆成图片。这样前端好加载一些。上传者也可以分成几次上传同一本书

**Reply by PirateBook1** [reply]
事实上只有图片集一种资料形式，到底是不是 pdf 其实是无所谓的

**Reply by banned-historical-archives** [reply]
为了溯源还是保留pdf吧

**Reply by banned-historical-archives** [reply]
保留pdf也没有太大影响，只有对比原始文件的时候会加载pdf，一般pdf也不会太大

**Reply by banned-historical-archives** [reply]
https://github.com/banned-historical-archives/banned-historical-archives.github.io/issues/442

**Reply by PirateBook1** [reply]
那么一本书一定要一次弄完吗。比如说《红旗》 一本杂志里就有着许多文章

**Reply by banned-historical-archives** [reply]
是的
